src/Cake.Core/ICakeArguments.cs

```diff
@@ -1,19 +1,11 @@
-﻿using System.Collections.Generic;
-
-namespace Cake.Core
+﻿namespace Cake.Core
 {
     /// <summary>
     /// Represents arguments passed to script.
     /// </summary>
     public interface ICakeArguments
     {
         /// <summary>
-        /// Initializes the argument list.
-        /// </summary>
-        /// <param name="arguments">The arguments.</param>
-        void SetArguments(IDictionary<string, string> arguments);
```

Comment:
Just note to self, potential breaking change, don't think anyone has used this externally though.